### PR TITLE
Fix: Address deprecation warning

### PR DESCRIPTION
**Pull request by yo1995**
This PR fixes the deprecation warning in `ReadSymbolsFromMobileStyleSymbolSettingsViewController`.

### CODE REVIEW

File: arcgis-ios-sdk-samples/Display information/Read symbols from a mobile style/ReadSymbolsFromMobileStyleSymbolSettingsViewController.swift [hunk 1]
```diff
@@ -120,17 +120,20 @@ class ReadSymbolsFromMobileStyleSymbolSettingsViewController: UITableViewControl
         } else if imageOperations[indexPath] != nil {
             return nil
         } else {
-            let operation = searchResult.symbol?.createSwatch(withWidth: 40, height: 40, screen: .main, backgroundColor: nil) { [weak self] (image, error) in
+            searchResult.symbol { [weak self] symbol, error in
```
Comment (sbaskaran) [inline arcgis-ios-sdk-samples/Display information/Read symbols from a mobile style/ReadSymbolsFromMobileStyleSymbolSettingsViewController.swift:123]: This error hasn't been handled, has it?

Comment (yo1995) [reply arcgis-ios-sdk-samples/Display information/Read symbols from a mobile style/ReadSymbolsFromMobileStyleSymbolSettingsViewController.swift:123]: Nope. 😳  
I assume the previous API will give nil when an error happens, and it doesn't proceed with a nil. So I just ignored the error in the new API. Should it be handled?

Comment (sbaskaran) [reply arcgis-ios-sdk-samples/Display information/Read symbols from a mobile style/ReadSymbolsFromMobileStyleSymbolSettingsViewController.swift:123]: I think so

Comment (yo1995) [reply arcgis-ios-sdk-samples/Display information/Read symbols from a mobile style/ReadSymbolsFromMobileStyleSymbolSettingsViewController.swift:123]: Fixed.

**Review by sbaskaran — APPROVED** [review]

**Review by philium — APPROVED** [review]

**Pull request merged**